charts/kubernetes-agent/templates/tentacle-pvc.yaml

```diff
@@ -8,7 +8,12 @@ metadata:
 spec:
   accessModes:
     - ReadWriteMany
-  storageClassName: {{ .Values.persistence.storageClassName | quote }}
+  {{- with .Values.persistence.storageClassName }}
+  storageClassName: {{ . | quote }}
```

Comment:
The customer will still need to define a storage class name to allow the PVC/PV to bind correctly if they provide a volumeName. The accessModes would also need to match. We _may_ be able to set the storageClassName to `""` to allow it to bind, but I'm not sure. Let me check the code.

Reply:
So do we need to also expose the `accessModes`?

Reply:
It's probably worth talking about whether we do or not